src/Plugin.Maui.Audio/AudioPlayer.macios.cs

```diff
@@ -96,6 +102,7 @@ public void Stop()
     bool PreparePlayer()
     {
         player.FinishedPlaying += OnPlayerFinishedPlaying;
+		player.EnableRate = true;
```

Comment:
Does it have any implications of having this on when someone is not using it? Should we turn it off if rate == 1?

Reply:
According the Apple Dev Doc, this needs to be set before call to PrepareToPlay method. I can not find any information that it will have any implications if turned on. To turn it on, only when needed will probably require to recreate player, set property and call PrepareToPlay. One solution could be that we add a optional SpeedActive property to the CreatePlayer method and only attempt to set speed if active on all platforms.